docs/FAQ.md

```diff
@@ -357,3 +358,38 @@ When you send this command, a new cell will be appended to the notebook with the
 <img  alt="image" src="https://user-images.githubusercontent.com/547415/210672882-9825764b-f2dc-4f57-9d9e-21cefc86bed5.png" width="60%">
 
 This command is not currently supported in other notebook frontends such as JupyterLab.
+
+## Troubleshooting
+
+### A cell runs forever
+
+It common for a notebook's kernel to get into a stuck state. Maybe you tried to load too much data or accidentally ran code containing an infinite loop. This happens often enough that a way to restart the kernel is a feature of most notebook providers.
+
+In Polyglot Notebooks, you can restart the kernel by running the `Polyglot Notebook: Restart the current notebook's kernel` command from the Command Palette. Note that atfer you do this, you'll need to rerun the notebook's cells, including reloading packages using `#r`.
+
+### Nothing happens when running a cell
+
+Sometimes VS Code updates have been applied or are pending and the Polyglot Notebooks extension needs to be updated. If things aren't working, here are a few things to check.
+
+If you see the following badge on the settings icon in the lower left corner of VS Code, it means there might be an update pending:
+
+<img width="48" alt="image" src="https://user-images.githubusercontent.com/547415/224158533-c01456c5-0759-46ac-a8c1-317586974a16.png">
+
+When you click it and see a `Restart to Update` message in the menu, then VS Code needs an update.
+
+<img width="299" alt="image" src="https://user-images.githubusercontent.com/547415/224158995-0d5864cc-57b3-416a-95bb-3d18ea42c7bb.png">
+
+You might also see, including after a VS Code update has been applied, that the Polyglot Notebooks extension requires a reload.
+
+<img width="360" alt="image" src="https://user-images.githubusercontent.com/547415/224161370-1c628967-ae0e-42b2-9c64-e3c1d7756f0b.png">
+
+If you're still seeing issues running code after checking for updates, please [open an issue](https://github.com/dotnet/interactive/issues/new/choose). 
```

Comment:
Here's couple of other interesting steps that I had jotted down. Would any of these be useful to add to this doc (under a separate header)?

* Is the issue you are encountering a bug with the currently opened notebook only or does it also repro within any notebook?
* Does it continue to repro if you start from a clean slate (i.e., close all documents and restart VS code)? (Since we have seen some bugs where state can bleed across notebooks...)
* Does it repro in stable v/s insiders v/s main?

Reply:
These sound a little more like bug investigation steps. The focus in the FAQ is on how people can unblock themselves. If they do hit a bug, I don't want to ask them to troubleshoot too deeply until they file an issue, because we often understand the details after the first report.